doc/usermanual/lua/lua_api.xml

```diff
@@ -3574,27 +3575,33 @@ NOTE2: If the parameter is a directory the call is non-blocking; the film object
 
 <varlistentry id="darktable_database_move_image_film"><term>film</term><listitem>
 <synopsis><link linkend="types_dt_lua_film_t">types.dt_lua_film_t</link></synopsis>
-<para>The film to move to</para>
+<para>The film to move to. To rename a file within the same film, set this to the image's current film.</para>
 
 </listitem></varlistentry>
 
+<varlistentry id="darktable_database_move_image_newname"><term>newname</term><listitem>
+<synopsis>string</synopsis>
+<para><emphasis>(Optional)</emphasis> If provided, rename the file with this new name. Must not contain any path separator characters.</para>
+
+</listitem></varlistentry>
 </variablelist>
 </section>
 
-<section status="final" id="darktable_database_copy_image">
+<section status="draft" id="darktable_database_copy_image">
 <title>darktable.database.copy_image</title>
 <indexterm>
 <primary>Lua API</primary>
 <secondary>copy_image</secondary>
 </indexterm>
 <synopsis>function( 
 	<emphasis><link linkend="darktable_database_copy_image_image">image</link></emphasis> : <link linkend="types_dt_lua_image_t">types.dt_lua_image_t</link>, 
-	<emphasis><link linkend="darktable_database_copy_image_film">film</link></emphasis> : <link linkend="types_dt_lua_film_t">types.dt_lua_film_t</link>
+	<emphasis><link linkend="darktable_database_copy_image_film">film</link></emphasis> : <link linkend="types_dt_lua_film_t">types.dt_lua_film_t</link>,
+	<emphasis><link linkend="darktable_database_copy_image_newname">newname</link></emphasis> : string
```

Comment:
enclose this line in brackets  i.e.

`[<emphasis><link linkend="darktable_database_copy_image_newname">newname</link></emphasis> : string]`

to denote that the argument is optional